gcovr/tests/subfolder-includes/reference/gcc-5/coverage.lib.h.4acb6a61176c15c0bede65bfbcd51fe2.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <title>GCC Code Coverage Report</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="coverage.css">
+  </head>
+
+  <body>
+    <h1>GCC Code Coverage Report</h1>
+
+    <hr/>
+
+    <div class="summary">
+<div>
+  <table class="legend">
+    <tr>
+      <th scope="row">Directory:</th>
+      <td>subfolder/subfolder/</td>
+    </tr>
+    <tr>
+      <th scope="row">File:</th>
+      <td>subfolder/subfolder/lib.h</td>
+    </tr>
+    <tr>
+      <th scope="row">Date:</th>
+      <td>2020-12-24 15:03:34</td>
+    </tr>
+  </table>
+</div>
+<div>
+  <table class="coverage">
+    <tr>
+      <th></th>
+      <th scope="col">Exec</th>
+      <th scope="col">Total</th>
+      <th scope="col">Coverage</th>
+    </tr>
+    <tr>
+      <th scope="row">Lines:</th>
+      <td>2</td>
+      <td>2</td>
+      <td class="coverage-high">100.0%</td>
+    </tr>
+    <tr>
+      <th scope="row">Branches:</th>
+      <td>0</td>
+      <td>0</td>
+      <td class="coverage-unknown">-%</td>
+    </tr>
+  </table>
+</div>
+    </div>
+
+    <hr/>
+
+<div class="file-source">
+  <table>
+    <tr>
+      <th align="right">Line</th>
+      <th align="right">Branch</th>
+      <th align="right">Exec</th>
+      <th align="left">Source</th>
+    </tr>
+
+
```

Comment:
My suggestion was to add something like this instead of a empty html:
```html
    <tr class="source-line">
      <td class="lineno">-</td>
      <td class="linebranch">
      </td>
      <td class="linecount"></td>
      <td class="src coveredLine">!!! File not found !!!</td>
    </tr>
```